### DISCUSSION TITLE
How to use **LeaveOnEmpty_Queue** function? Like where i have to put this??...

### DISCUSSION THREAD

**Discussion by Noman17-tect** [SudhanPlayz/Discord-MusicBot#664] [Q&A]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Eg. I'm always frustrated when [...]

**Describe the ideal solution**

**Describe alternatives you've considered**

**Additional context**

**Comment by captainion2119** [comment]
... empty request smh.

**Comment by darrenuselinux** [comment]
uhh what
